### DISCUSSION TITLE
AWS SDK Credentials Getting Overwritten

### DISCUSSION THREAD

**Discussion by haydenwade** [vercel/community#1386] [Help] [answered]
### Summary

I'm using AWS Rekognition SDK but unable to get my application to authenticate due to the following error:

`UnrecognizedClientException: The security token included in the request is invalid.`

Relavant Code:
```
const {
  RekognitionClient,
} = require("@aws-sdk/client-rekognition"); //    "@aws-sdk/client-rekognition": "^3.258.0"


this.client = new RekognitionClient({
      region: 'us-east-1',
      credentials: {
        accessKeyId: process.env.AWS_REK_ACCESS_KEY_ID,
        secretAccessKey: process.env.AWS_REK_SECRET_ACCESS_KEY,
      },
    });

this.client.send(command);
```

I made sure not to use the [reserved environment variables here](https://vercel.com/docs/concepts/limits/overview#reserved-variables). 
The credentials I have configured on my project in vercel work locally just fine.
I confirmed via log messages that the credentials were indeed correct when instantiating the client.

Only thing i could think of causing this is that `@aws-sdk/client-rekognition` library is taking the reserved variables over the ones passed into the constructor. I have not been able to verify this in code yet.

Any ideas for a solution here?

**Comment by smaeda-ks** [comment, marked as answer]
This should now be possible: https://vercel.com/changelog/march-2023
